python/src/equistore/block.py

```diff
@@ -229,6 +279,34 @@ def add_gradient(
         """
         Add a set of gradients with respect to ``parameters`` in this block.
 
+        Here is an example of how to use this method:
```

Comment:
Seems to fix the tox documentation tests
```suggestion
        Here is an example of how to use this method:
        
```